x/datadeal/keeper/deal_test.go

```diff
@@ -174,13 +174,22 @@ func (suite *dealTestSuite) TestSellDataSuccess() {
 
 	dataSale, err := suite.DataDealKeeper.GetDataSale(suite.Ctx, suite.dataHash1, uint64(1))
 	suite.Require().NoError(err)
-	suite.Require().Equal(dataSale.VerifiableCid, suite.verifiableCID1)
 
+	suite.Require().Equal(dataSale.VerifiableCid, suite.verifiableCID1)
 	suite.Require().Equal(dataSale.DealId, uint64(1))
 	suite.Require().Equal(dataSale.VerificationVotingPeriod, suite.OracleKeeper.GetVotingPeriod(suite.Ctx))
 	suite.Require().Equal(dataSale.Status, types.DATA_SALE_STATUS_VERIFICATION_VOTING_PERIOD)
 	suite.Require().Equal(dataSale.SellerAddress, suite.sellerAccAddr.String())
 	suite.Require().Equal(dataSale.DataHash, suite.dataHash1)
+
+	events := suite.Ctx.EventManager().Events()
+
+	suite.Require().Equal(3, len(events[0].Attributes))
+	suite.Require().Equal(types.EventTypeDataVerificationVote, events[0].Type)
+	suite.Require().Equal(types.AttributeKeyVoteStatus, string(events[0].Attributes[0].Key))
+	suite.Require().Equal(types.AttributeValueVoteStatusStarted, string(events[0].Attributes[0].Value))
+	suite.Require().Equal(types.AttributeKeyDealID, string(events[0].Attributes[1].Key))
+	suite.Require().Equal(types.AttributeKeyDataHash, string(events[0].Attributes[2].Key))
```

Comment:
Please refer the previous tests for event.
If some event is added, the order might be changed. So I suggest this way.

https://github.com/medibloc/panacea-core/blob/38cb583403c3a81fd9a6bf1484cf67bd0e4dc132/x/datadeal/abci_test.go#L237

Reply:
Yes, I saw this test code. But the there has an only one event type in `SellData`. So I just wrote like this.

Reply:
Ok. Let's keep this way, then.